install/install.sh

```diff
@@ -53,11 +53,43 @@ then
   usage;
 fi
 
+if grep -q -i "release 6" /etc/redhat-release || grep -q -i "release 7" /etc/redhat-release
```

Comment:
You can simplify this logic, something like:

```sh
if dnf --version; then
    PKG_MGR=dnf
else
    PKG_MGR=yum
fi
```

Later in the code you can just call:

```sh
sudo "${PKG_MGR}" install -y ansible
```

Reply:
Or you could use `command -v dnf` like you did below for ansible